services/src/main/java/com/linkedin/kmf/services/configs/ConsumeServiceConfig.java

```diff
@@ -42,6 +42,10 @@
   public static final String CONSUMER_PROPS_CONFIG = "consume.consumer.props";
   public static final String CONSUMER_PROPS_DOC = "The properties used to config consumer in consume service.";
 
+  public static final String LATENCY_SLA_MS_CONFIG = "consume.latency.sla.ms";
```

Comment:
Yeah, the configuration name is a little weird. It may be better to change all the `consume.consumer.*`  to `consumer.*`